data/strings/types_strings.txt

```diff
@@ -23312,7 +23065,7 @@
     zh-Hant = 家電店
 
   [type.shop.art]
-    en = Arts Shop
+    en = Art Shop
```

Comment:
It's "Art Gallery" if you buy art, and "Art Supplies Store" (e.g. Michael's) if you buy stuff to do your own art or frame other art.

Reply:
art gallery is too hard to distinguish with art galleries of the museum type, maybe  'Artwork Shop' will do?